### ISSUE TITLE
Label has unused export property

### ISSUE THREAD

**Issue by lovasoa** [bestguy/sveltestrap#111]
Thank you for this library. When trying the example from the README, I get warnings : 

![image](https://user-images.githubusercontent.com/552629/72218354-7e350280-353a-11ea-877c-17d79da3671b.png)

```
Label has unused export property 'xs'. If it is for external reference only, please consider using `export const 'xs'` (Label.svelte:20:13)

Label has unused export property 'sm'. If it is for external reference only, please consider using `export const 'sm'` (Label.svelte:21:13)

Label has unused export property 'md'. If it is for external reference only, please consider using `export const 'md'` (Label.svelte:22:13)

Label has unused export property 'lg'. If it is for external reference only, please consider using `export const 'lg'` (Label.svelte:23:13)

Label has unused export property 'xl'. If it is for external reference only, please consider using `export const 'xl'` (Label.svelte:24:13)

Label has unused export property 'widths'. If it is for external reference only, please consider using `export const 'widths'` (Label.svelte:25:13)
```

**Comment by frederikhors** [comment]
Already open PR: https://github.com/bestguy/sveltestrap/pull/88

**Comment by bestguy** [comment]
Hi @lovasoa , this should be corrected in sveltestrap@3.3.2
